proto/stream_plan.proto

```diff
@@ -580,8 +590,10 @@ message StreamCdcScanNode {
   // The rate limit for the stream cdc scan node.
   optional uint32 rate_limit = 6;
 
-  // Whether skip the backfill and only consume from upstream.
-  bool disable_backfill = 7;
+  reserved 7;
+  reserved "disable_backfill";
```

Comment:
Breaking change. Considering that we have published in [document](https://docs.risingwave.com/docs/current/ingest-from-mysql-cdc/), recommend keeping the field for now and making the code compatible with old streaming plan e.g. when `options` is not present, use this one.